src/java.base/share/classes/sun/security/ssl/SignatureScheme.java

```diff
@@ -375,10 +387,32 @@ static List<SignatureScheme> getSupportedAlgorithms(
             List<ProtocolVersion> activeProtocols) {
         List<SignatureScheme> supported = new LinkedList<>();
 
-        List<SignatureScheme> schemesToCheck =
-                config.signatureSchemes == null ?
-                    Arrays.asList(SignatureScheme.values()) :
-                    namesOfAvailable(config.signatureSchemes);
+        // If config.signatureSchemes is non-empty then it means that
+        // it was defined by a System property.  Per
+        // SSLConfiguration.getCustomizedSignatureScheme() the list will
+        // only contain schemes that are in the enum.
+        // Otherwise, use the enum constants (converted to a List).
```

Comment:
I think you mean "non-null" (not "non-empty"); I don't think "system" or "list" deserve to be capitalized here.

Reply:
Updated comment to be `non-null` and also lower case `list`

Reply:
Please use lower case for `system`. Could you also specify which system property is involved here instead of just saying "a system property"?

Reply:
Used lower case for system. Also added the two property names that can be in use here for setting signature schemes with TLS.